\documentclass[12pt]{article}
\usepackage[utf8]{inputenc}
\usepackage{xcolor}
\usepackage[numbers, compress]{natbib}
\bibliographystyle{unsrtnat}
\usepackage{chapterbib}
\usepackage[a4paper, portrait, margin=2cm]{geometry}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage{lineno}
\usepackage[hidelinks]{hyperref}
\usepackage[toc,page]{appendix}

\newcommand{\igjindent}{\,\,\,\,\,\,\,\,\,}
\newcommand{\be}{\begin{equation}}
\newcommand{\ee}{\end{equation}}
\newcommand{\rcgtcr}{\textcolor{red}}
\newcommand{\rcgtcb}{\textcolor{black}}

\linenumbers 

\title{Mitochondrial network structure controls cell-to-cell mtDNA variability generated by cell divisions}
\author{Robert C. Glastad${}^1$, Iain G. Johnston${}^{1,2,*}$ \\ \footnotesize ${}^1$ Department of Mathematics, University of Bergen, Bergen, Norway \\ \footnotesize ${}^2$ Computational Biology Unit, University of Bergen, Bergen, Norway \\ \footnotesize ${}^*$ correspondence to \url{iain.johnston@uib.no}}
\date{}

\begin{document}

\maketitle
\begin{abstract}
Mitochondria are highly dynamic organelles, containing vital populations of mitochondrial DNA (mtDNA) distributed throughout the cell. Mitochondria form diverse physical structures in different cells, from cell-wide reticulated networks to fragmented individual organelles. These physical structures are known to influence the genetic makeup of mtDNA populations between cell divisions, but their influence on the mtDNA quality control remains less understood.
\end{abstract}

\section*{Introduction}
DNA repair is ubiquituous throughout the eukaryotes, and is central to keep any kind of DNA in working order. These mechanisms are characterized by specific genes, encoding protein that interact with DNA and/or other molecules in such a way as so reconstitute the original copy of DNA XXX CITE. Mitochondria are vital bioenergetic organelles responsible for the majority of the eukaryotic cell's energy production. Due to their evolution from within the $\alpha$proteobacteria cite XXX, mitochondria retain small genomes (mtDNA) that are central to their energy producing capability.

Although one might have expected that mtDNA is \textit{more} protected than is nuclear DNA, the opposite is in fact the case. Moreover, depending upon cellular demand, mtDNA mutation rates (nucleotide substitution) tend to be far larger in the mitochondria, likely owing to frequent replication on imperfect ribosomes within the mitochondria. To remain in working order, then, it is vital that mitochondria repair their mtDNA molecules that if they are incorrect, and if they cannot do so, discard the mtDNA entirely -- with its host mitochondrion. However, no biological process is perfect, and thus, mutated mtDNA molecules cannot always be removed from the system. In the long run, these mutated mtDNA molecules may wreak havoc on the system's proper functioning, and even escape copy number control \citep{pereira2021cellular}.

However, before then, the cell has a number of tricks, outside of mtDNA repair, up its proverbial sleeve as well. \citep{bereiter1996dynamics}

For instance, mtDNA molecules are known to complement each other when they reside within the same mitochondria. Moreover, in many cell types, mitochondrial dynamics allow the population of mitochondria to exchange matrix contents if the complete process remains short in length ("kiss-and-run" dynamics) \citep{logan2010dynamic}, but also inner membrane components are exchanged when the fused state persists over longer timescales (cite XXX). This process can essentially mask mutated mtDNA molecules. Such fission and fusion processes may be undertaken up to 5 times per cell cycle \citep{twig2008mitochondrial}.

What is more, one may speculate that the demand placed upon singular wildtype mtDNA is larger whenever that molecule has to support a mitochondrion on its own. Thus mutation rates may depend upon whether or not a wildtype mtDNA molecule "cooperates" with other wildtype mtDNA molecules. This effect is

\section*{Results}

\section*{Discussion}

\section*{Acknowledgments}
This project has received funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (Grant agreement No. 805046 (EvoConBiO) to IGJ).

\bibliography{refs.bib}

\clearpage
\newpage

\begin{appendices}
\section{Models of mtDNA copy number variance}

\end{appendices}
\end{document}
